cmd/kcp-catalog/main.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2021 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	goflags "flag"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/component-base/cli"
+	"k8s.io/component-base/version"
+	"k8s.io/klog/v2"
+
+	"github.com/kcp-dev/kcp/pkg/cmd/help"
+)
+
+func main() {
+	cmd := &cobra.Command{
+		Use:   "kubectl-kcp",
```

Comment:
Copy/paste error?

Reply:
I have modified this to be `kubectl-catalog`, to be a bit more generic. Is this fine, or would we like to have `kcp-catalog` or `kubectl-kcp catalog`. The whole command would look like `<placeholder> bind catalogentry root:catalog:catalogentry-sample`